libpf/trace.go

```diff
@@ -10,15 +10,27 @@ package libpf
 // stack frame via the file ID and line number at the offset i in the trace. The
 // information for the most recently called function is at offset 0.
 type Trace struct {
-	Files      []FileID
-	Linenos    []AddressOrLineno
-	FrameTypes []FrameType
-	Hash       TraceHash
+	Files              []FileID
+	Linenos            []AddressOrLineno
+	FrameTypes         []FrameType
+	MappingStart       []Address
+	MappingEnd         []Address
+	MappingFileOffsets []uint64
+	Hash               TraceHash
 }
 
 // AppendFrame appends a frame to the columnar frame array.
```

Comment:
nit: as this is a public API element of the package we could be more explicit.
```suggestion
// AppendFrame appends a frame to the columnar frame array without mapping information.
```